Client/Src/Common/timer-overlay.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+import React, {
+  StyleSheet,
+  Component,
+  View,
+  Text,
+  TouchableOpacity
+} from 'react-native';
+
+// importing styles
+const styles = StyleSheet.create(require('../styles.js'));
+const timerMixin = require('react-timer-mixin');
+const Timer = require('../components/Timer')
+var moment = require('moment');
+var endTime;
+
+var TimerOverlay = React.createClass({
+  getInitialState: function(){
+   return {
+      timeRemaining:5000
+    } 
+  },
+  componentDidMount: function() {
+    // WHAT WE HAVE: ETA + ACCEPTABLE DELAY
+    timerMixin.setInterval.call(this, function(){
+      this.setState({timeRemaining: this.state.timeRemaining -= 1000});
+    },1000);
+  },
+  renderOverlay:function(){
+    if(this.state.timeRemaining < 0){
+      return componentStyles.overTime
+    } else {
+      return componentStyles.background
+    }
+  },
+  render: function() {
+    return (
+      <View style={this.renderOverlay()}>
+        <View style={componentStyles.alertContainer}>
+        <Timer timeRemaining={this.state.timeRemaining} />
+        </View>
+      </View>
+    );
+  }
+
+});
+
+module.exports = TimerOverlay;
+
+import { Dimensions } from 'react-native';
+  var width = Dimensions.get('window').width; //full width
```

Comment:
Width and Height can be destructured off DImensions.

const {width, height} = Dimensions.get('window');